docs/CONTRIBUTOR_GUIDE.md

```diff
@@ -17,6 +17,25 @@ git clone https://github.com/kubernetes-sigs/krew .
 git remote set-url origin --push no_push   # to avoid pushes
 ```
 
+### Mac specific requirements
+
+The tools provided in the `hack` folder expect you to use GNU binaries.
+
+The easiest way to install them is to use brew
```

Comment:
you can merge

Reply:
I merged them into one sentence. hope that's what you where refering to.